python/plugins/processing/algs/saga/SagaAlgorithmProvider.py

```diff
@@ -82,7 +82,7 @@ def setActive(self, active):
 
     def canBeActivated(self):
         version = SagaUtils.getInstalledVersion(True)
-        if version is not None and version.startswith(REQUIRED_VERSION):
+        if version is not None and version > REQUIRED_VERSION:
```

Comment:
```suggestion
        if version is not None and version >= REQUIRED_VERSION:
```

Reply:
Sorry for the late response.
On FreeBSD, the latter patch makes newer versions, like the recent 7.2.0 usable with the FreeBSD port of QGIS, see under https://www.freshports.org/graphics/qgis. 

Of course, some of the newer SAGA GIS modules are not documented within QGIS processing until now, but on the other hand, many many bug fixes of newer SAGA GIS versions are included.